reporter-web-app/.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
commit-msg: Would it make sense to provide a rationale for choosing`10 minutes` as value?

Reply:
Please see the link in the commit message; TL;DR it's arbitrarily high.

Reply:
I read the link and decided to stop at 'Here I set it to 10 minutes for OVERKILL'.

Reply:
> I read the link and decided to stop at 'Here I set it to 10 minutes for OVERKILL'.

I didn't read any further either :laughing:

Reply:
But I'll amend the commit message...

Reply:
I decided that at least mentioning the default timeout makes sense, and added that to the commit message, too.